Admin/BaseFieldDescription.php

```diff
@@ -415,20 +415,6 @@ public function getMappingType()
     }
 
     /**
-     * Camelize a string.
-     *
-     * @static
-     *
-     * @param string $property
-     *
-     * @return string
-     */
-    public static function camelize($property)
```

Comment:
This is BC break.

If this method is really not needed, it should be deprecated instead.